android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java

```diff
@@ -128,11 +128,9 @@ public void handleCreationDict(KrollDict d)
 		if (d.containsKey(TiC.PROPERTY_GROUP_KEY)) {
 			setGroupKey(TiConvert.toString(d, TiC.PROPERTY_GROUP_KEY));
 		}
-		/*
 		if (d.containsKey(TiC.PROPERTY_GROUP_ALERT_BEHAVIOR)) {
 			setGroupAlertBehavior(TiConvert.toInt(d, TiC.PROPERTY_GROUP_ALERT_BEHAVIOR));
 		}
-		*/
```

Comment:
@garymathews Just to be sure: This needs to be uncommented in 6_2_X? This was part of the merge-conflicts.